.github/workflows/containerd.yml

```diff
@@ -344,7 +344,7 @@ jobs:
       - name: Upload containerd ${{matrix.version}} logs
         uses: actions/upload-artifact@v1
         with:
-          name: containerd-${{matrix.version}}-${{ matrix.os }}-${{matrix.runtime}}-${{matrix.runc}}-${{github.sha}}.log
+          name: containerd-${{ replace(matrix.version, '/', '_') }}-${{ matrix.os }}-${{matrix.runtime}}-${{matrix.runc}}-${{github.sha}}.log
```

Comment:
I found examples of replace on GitHub: https://github.com/search?type=code&auto_enroll=true&q=path%3A*.yaml+%22%24%7B%7B+replace%22 but didn't find any documentation. Any hints are appreciated.